packages/styles/scss/components/tableofcontents/_tableofcontents.scss

```diff
@@ -237,15 +237,9 @@ $hover-transition-timing: 95ms;
   }
 
   .#{$prefix}--tableofcontents-horizontal__contents {
-    a[name] {
+    [name] {
       display: block;
-      color: $text-01;
-      transform: translateY(calc(-1 * #{$layout-03}));
-      padding-top: 1rem;
-
-      > * {
-        transform: translateY($layout-03);
-      }
+      padding-top: 3rem;
```

Comment:
@mwanberg One more small thing, can we use spacing token here? Thanks!

Reply:
Done!